popup.js

```diff
@@ -36,13 +36,24 @@ chrome.storage.sync.get('leethub_token', (data) => {
           chrome.storage.sync.get('mode_type', (data2) => {
             if (data2 && data2.mode_type === 'commit') {
               $('#commit_mode').show();
-              /* Get problems solved count */
-              chrome.storage.sync.get('stats', (psolved) => {
-                const { stats } = psolved;
-                if (stats && stats.solved) {
-                  $('#p_solved').text(stats.solved);
-                }
-              });
+              /* Get problem stats and repo link */
+              chrome.storage.sync.get(
+                ['stats', 'repo'],
```

Comment:
why not have repo as object within stats?

Reply:
Good point. I can combine.